src/aks-preview/azext_aks_preview/_completers.py

```diff
@@ -63,12 +63,19 @@ def get_vm_sizes(cli_ctx, location):
 
 
 @Completer
-def get_ossku_completion_list(cmd, prefix, namespace, **kwargs):  # pylint: disable=unused-argument
-    """Return the list of allowed os-sku values"""
+def get_cluster_ossku_completion_list(cmd, prefix, namespace, **kwargs):  # pylint: disable=unused-argument
+    """Return the list of allowed os-sku values when create cluster"""
 
     return ["Ubuntu", "CBLMariner"]
 
 
+@Completer
+def get_nodepool_ossku_completion_list(cmd, prefix, namespace, **kwargs):  # pylint: disable=unused-argument
+    """Return the list of allowed os-sku values when add nodepool"""
+
+    return ["Ubuntu", "CBLMariner", "Windows2019", "Windows2022"]
```

Comment:
It would be nice if you could replace these strings with the consts. Also the ones in get_cluster_ossku_completion_list.

Reply:
Good guidance. Updated.